docs/guides/operations-guide/openstack/flavor-manager.md

```diff
@@ -4,3 +4,122 @@ sidebar_position: 51
 ---
 
 # Flavor Manager
+
+## Overview
+
+The openstack-flavour-manager operates as a facilitator that orchestrates VM compute flavors in alignment 
+with the OSISM OpenStack cloud standard, utilizing standardized YAML files supplied by OSISM. 
```

Comment:
Does "OSISM OpenStack cloud standard" make sense here? Because we are probably focusing more on SCS or flavor standards in general here, right?

Reply:
Nah. Use always SCS standard when talking about our standards.

Reply:
Has been corrected